dot/rpc/modules/test_data/test_metadata.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package testdata
+
+import (
+	"archive/zip"
+	"bytes"
+	_ "embed"
+	"io"
+)
+
+//go:embed test_metadata.zip
+var testMetaData []byte
+
+func init() {
+	bytesReader := bytes.NewReader(testMetaData)
+	zipReader, err := zip.NewReader(bytesReader, int64(bytesReader.Len()))
+	if err != nil {
+		return
```

Comment:
You should panic when an error occurs, otherwise it'll be hard to understand why the test is failing.